.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits until the commit message
+# contains [ci skip]
+# Enable 'Do not build on "Push" events' in the AppVeyor project settings
+# to only build commits from pull requests
+branches:
+  only:
+    - master
+
+only_commits:
+  files:
+    - content/
+
+image: ubuntu
+services:
+  - docker
+
+install:
+  - source ci/install.sh
```

Comment:
Ah I wonder if we `source` rather than bash on travis, we could get each command to be in the log?